src/components/TeamCard/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './style.less';
+
+interface Props {
+  pic: string;
+  member: string;
+  role: string;
+}
+const TeamCard: React.FC<Props> = ({pic, member, role}) => {
+  return (
+    <div className="TeamCard">
+      <img className="image "src={pic} alt="Avatar"/>
```

Comment:
Nice you're adding alts - maybe use a formatted string for it like `${member-name} photo`